src/diem/testing/miniwallet/app/offchain_api_v2.py

```diff
@@ -102,6 +116,43 @@ def _handle_offchain_payment_command(self, sender_address: str, request: Command
             account_id = self.app.store.find(Subaddress, subaddress_hex=subaddress).account_id
             self._create_payment_command(account_id, new_offchain_cmd, validate=True)
 
+    def _handle_offchain_reference_i_d_command(
+        self, request_sender_address: str, request: offchain.CommandRequestObject
+    ) -> Dict[str, Any]:
+        if request.command_type != CommandType.ReferenceIDCommand:
+            raise protocol_error(
+                ErrorCode.unknown_command_type,
+                f"unknown command_type: {request.command_type}",
+                field="command_type",
+            )
```

Comment:
Remove above if check, the function is never called if the command type does not match the type